front/app/modules/commercial/insights/services/insightsDetectCategories.ts

```diff
@@ -0,0 +1,22 @@
+import { API_PATH } from 'containers/App/constants';
+import streams, { IStreamParams } from 'utils/streams';
+
+const getInsightsDetectCategoriesEndpoint = (viewId: string) =>
+  `insights/views/${viewId}/detected_categories`;
+
+export type IInsightsDetectedCategoriesData = { names: string[] };
```

Comment:
Great, thanks for the info! I think I will just wait for you to merge it so that we can test the integration directly and make sure everything works well and there are no more changes to be done :)